src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+:root {
+  --header-height: 60px;
+  --underline-height: 4px;
+  --horizontal-padding: 50px;
+  --hover-color: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  width: calc(100% - 2 * var(--horizontal-padding));
```

Comment:
You dont need to set this width property, as this container will take 100% of width by default